arrow-site/doc/docs/arrow-core/datatypes/either/README.md

```diff
@@ -269,10 +269,10 @@ y.fold({ 1 }, { it + 3 })
 ```
 
 The `getOrHandle()` operation allows the transformation of an `Either.Left` value to a `Either.Right` using
-the value of `Left`. This can be useful when a mapping to a single result type is required like `fold()` but without
+the value of `Left`. This can be useful when mapping to a single result type is required like `fold()`, but without
```

Comment:
like with `fold()`

(Feel free to ignore any of these: I'm just making suggestions.)